### ISSUE TITLE
[BUG] set_position(&mut self, val: i32) is always returning an error

### ISSUE THREAD

**Issue by nicolasbauw** [MoAlyousef/fltk-rs#288]
**Describe the bug**
The set_position() function is always returning an error (with an output and input widget)

**To Reproduce**
```
let mut header_view = Output::default().with_size(WIN_W, 18).with_pos(0,0);
header_view.set_position(WIN_W/2 as i32).unwrap();
```

(no need to cast as i32 - same result whatever the value)

**Expected behavior**
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Internal(FailedOperation)'
(unwrap on purpose)

**Desktop (please complete the following information):**
 - OS: macOS 10.15
 - Version 0.8.7
 - rust 1.46 stable

**Comment by MoAlyousef** [comment]
Hi

I think the docs might be lacking here. 
InputExt::set_position() sets the position of the cursor within the input output widget, i.e. where the text is inserted. It would throw when the text insertion is out of bounds. 
If you want to reposition the widget, you can use WidgetExt::set_pos(x, y)
header_view.set_pos(x, y)

**Comment by nicolasbauw** [comment]
What I aimed was to center the text inside the output widget (which is as large as screen), so I guess it is the right function ?
Note that the error occurs before I insert any text in the widget.

**Comment by MoAlyousef** [comment]
You are right. It's strange that even when the operation succeeds FLTK returns 0, which is translated in the bindings as an error.
I'll look further into the FLTK docs tomorrow, and maybe inquire in the fltk forums.

**Comment by MoAlyousef** [comment]
So looking into this I found a bug in the wrapper which ignored the return value of set_position. So calling it on an empty input of output widget now should error out. 
For an empty input/output widget, you might have to append empty spaces to get the position you want. Currently FLTK doesn't support aligned input/output widgets. 
I have pushed an example here which draws something like an output widget and has aligned output:
https://github.com/MoAlyousef/fltk-rs/blob/master/fltk/examples/aligned_output.rs
The example aligns the text in the center.
